### PR TITLE
Javascript code updates

### PR DESCRIPTION
**Pull request by mrossi113**
This is the PR with all of the changes to the JavaScript and HTML files. The first commits were needed to view the changes in CodePipeline second commits will be JavaScript updates to the code.

Most of the changes occurred on this page:
http://custom-env.pac9mpkmm6.us-east-1.elasticbeanstalk.com/add-score.html

The links in the navigation and button link was added to this page:
http://custom-env.pac9mpkmm6.us-east-1.elasticbeanstalk.com/dashboard.html

### CODE REVIEW

**Comment by mrossi113** [comment]
@cruoti @todoubled711 @keeena 
Working on this just need to add JavaScript updates now.

**Comment by mrossi113** [comment]
@cruoti @todoubled711 @keeena 
All set just need to know where the submit button will be directed to for add score and the dashboard will need to pull scores from the database as well. The button submit need to update only at the moment. 

@todoubled711 The add score was changed a bit for the table was not working well when adding a score. Might need to add some padding or margin to move the holes over a bit.

**Comment by mrossi113** [comment]
Note I made the hole score required min 1 and max 100 if someone leaves it 0 or >100 it will not submit the score. I did not make the course name required or date which I can very easily if needed. The clear button will work on the course name, date, and each hole score will be reset now. It also accessible for tabbing and up and down arrows will change the values in the hole fields. The date field would take some time to make it more accessible. I think the only thing that is needed now is some padding on the holes and total fields.

**Comment by cruoti** [comment]
I have:
- I updated the holes to have a default score of 3.  
- I have also made the course a drop down from a pre determined set of courses.

- I have also been making the player dashboard based on pre determined scores as well.  This change has not been pushed.

**Comment by cruoti** [comment]
I just pushed my commits.  It has the the features mentioned above.  I will wait to see if it works with the Pipeline.  If so, then I will merge it with develop and switch the pipeline.  Then Kian will be able to switch the branch on the pipeline as necessary to test the DB.

File: course_data.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+var courses = ["Wickham", "Center Springs", "Vernon"];
+
+function getCourseNamesFromDB() {
+    /*
+    TODO: Replace with DB calls
+    return list of course names like `var courses`
+     */
+    return courses;
+}
```
Comment (cruoti) [inline course_data.js:13]: @keeena 
Look at the `getCourseNamesFromDB()` function.  Right now the code has a variable `courses` that contains a list of all the course names for which we can add scores.  The function returns that list.  

We would just need to add the DB calls in this function.  It doesn't really matter how you make the calls as long as you arrange the returned data to be a list of strings that are the course names.

I also suppose the DB call doesn't need to be in this file per se.  You could create another file with the DB functions and then just make a call to that function from within this one.

File: dashboard.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+function get_player_data_from_db(playername) {
+    /*
+    TODO: Replace with DB calls
+    Return hash like `var cruoti` is setup above
+     */
+    return playerData[playername];
+}
+
```
Comment (cruoti) [inline dashboard.js:50]: @keeena 

Look at `get_player_data_from_db()` function and the variable `playerData`.  `playerData` is a hash that has keys of the usernames and the values are the data for that player.  That data is a hash itself that where the key is an ID of the round and the value is another hash that contains the data, course name, and scores for each hole.  Look at variable `cruoti` to see what this looks like.

You can make the DB calls from within this function or move the calls to another file to be shared with the course DB calls.  Whichever way you set it up, the main takeaway is that this function needs to return the player data for a given player, like variable `cruoti`.

**Comment by cruoti** [comment]
It looks like it is working on Elastic Beanstalk.  I'm going to merge it into develop and change the branch for the Pipeline.

**Pull request merged**